### DISCUSSION TITLE
Markdown is not available once the files are converted into textBundle format

### DISCUSSION THREAD

**Discussion by rasivasu** [glushchenko/fsnotes#1317] [Q&A] [answered]
I have some text files where the markdown options are still available and I am able to preview the document.  But I am not getting markdown options for files of type textBundle.

![image](https://user-images.githubusercontent.com/1153582/157264072-946827f7-c589-4bba-870d-d0ed61a68ac5.png)

**Comment by rasivasu** [comment, marked as answer]
@glushchenko Didn't realize that you are from Ukraine; just read the previous post.  I am sorry to hear the news.

Others, I resolved the above mentioned issue by creating normal text instead of RTF.  Thanks.

**Comment by glushchenko** [comment]
I recommend to update app, many bugs in older versions.

**Reply by rasivasu** [reply]
Thank you.
